### ISSUE TITLE
Support ES2020 "export * as ..." syntax

### ISSUE THREAD

**Issue by FredKSchott** [guybedford/es-module-lexer#48]
`export * as ...` Syntax was added to ES2020: https://2ality.com/2019/12/ecmascript-2020.html

es-module-lexer doesn't break on this new syntax, but it also doesn't detect it either:

```
const {parse} = require('es-module-lexer');
parse(`export * as Function from '@tsfun/function');
// returns: [ [], [] ]
```

**Comment by guybedford** [comment]
Thanks for posting - sure, I've gone ahead and published  0.3.18 with this support in https://github.com/guybedford/es-module-lexer/commit/130610e987c66b7d0a3c0792f1a5f1fe3bdb5bac.

**Comment by FredKSchott** [comment]
Awesome!
